cartofacade/capi_mock.go

```diff
@@ -97,7 +97,7 @@ func (cf *CartoMock) getInternalState() ([]byte, error) {
 	return cf.GetInternalStateFunc()
 }
 
-// runFinalOptimization calls the injected GetInternalState or the real version.
+// runFinalOptimization calls the injected RunFinalOptimization or the real version.
```

Comment:
while we are here could we fic the above comments to be lowercase as well :)

Reply:
Are you referring to those starting with `Get`? I am leaving those out on purpose because I have a PR open that removes `Get` and have taken care of the comments there already. My motivation stems from not having to deal with rebase conflicts :P

Reply:
yes, fair enough!